Packages/Order Entry Results Reporting/Routines/ORQ20.m

```diff
@@ -88,6 +90,10 @@
  Q Y
  ;
 DATE(X) ; -- Return date formatted as 00/00/0000 00:00
+ I $G(DUZ("LANG"))>1 D  Q $$FMTE^XLFDT(X) ; OSE/SMH - date i18n date/time
+ . ; 12345678901234
```

Comment:
What are these comments trying to tell me?

Reply:
This was for me to figure out where the seconds are in a Timson date.

Reply:
if the comment is important to leave there can you add more to it to make that obvious?